cpp/src/community/k_truss_impl.cuh

```diff
@@ -656,265 +291,53 @@ k_truss(raft::handle_t const& handle,
                                                    *vertex_partition_range_lasts);
     }
     renumber_map = std::move(tmp_renumber_map);
+
```

Comment:
You can use masking instead of creating a graph except for the final DODG graph creation (this DODG graph will be used again & again so better create a new graph than using masking as masking has its own overhead).